example_site/src/components/ConfigurationControls.js

```diff
@@ -84,7 +97,10 @@ export default function ConfigurationControls({ isOpen }) {
     setAccount(null);
 
     // Correct orientation for editor
-    const finalOrientation = orientation ? "right" : "left";
+    let finalOrientation;
+    if (orientation===true) finalOrientation = 'right';
+    else if (!orientation) finalOrientation = 'left';
+    else finalOrientation = orientation; 
```

Comment:
This is getting tricky isn't it! A few things to pay attention to here:
1. It's tempting to leave out the closures (brackets) for these if statements, but in my opinion it does make this code block a little less readable, and makes it easier for errors to slip through.
2. Since your final else statement assigns `finalOrientation` to `orientation`, you could use that as the initialization value on line 100 instead of using the `else` statement.
3. I think that the real trickiness is coming from having our orientation value be in two different formats (true vs. false/right vs.left) so let's figure out a cleaner way to get this value!